tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -79,9 +79,11 @@ Your service endpoint region is selected by default. Have a quick check and clic
 
 ### Step 3. Create an AWS interface endpoint
 
-TiDB Cloud begins creating an endpoint service, which takes 3 to 4 minutes.
+> **Note:**
+>
+> For each Dedicated Tier cluster created after March 28, 2023, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.
 
-When the endpoint service is created, take a note of your endpoint service name from the command in the lower area of the console.
+If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait for 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.
```

Comment:
```suggestion
If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.
```